app/views/layouts/_navbar.haml

```diff
@@ -1,12 +1,28 @@
-%nav.navbar.navbar-expand-lg.mb-2.bg-light{aria: {label: 'Main'}}
-  - if @current_user
-    = navbar_link 'Passengers', passengers_path
-    = navbar_link 'Add New Passenger', new_passenger_path
-    = navbar_link 'Dispatch Log', log_index_path
-    = navbar_link 'Archived Passengers', archived_passengers_path
-  - if @current_user&.admin?
-    = navbar_link 'Manage Users', users_path
-    = navbar_link 'Manage Mobility Devices', mobility_devices_path
-  = navbar_link 'Site Feedback', feedback_index_path
-  = navbar_link 'Brochure', brochure_passengers_path
-  = link_to 'Logout', destroy_session_path, class: 'ml-auto mr-2 nav-link btn btn-secondary'
+%nav.navbar.navbar-expand-lg.navbar-light.bg-light{aria: {label: 'Main'}}
+  %ul.navbar-nav.mr-auto
+    - if @current_user
+      %li.nav-item.dropdown
+        = link_to 'Passengers', '#', class: 'nav-link dropdown-toggle', id: 'passenger-dropdown',
```

Comment:
Any though of trying to add some sort of signal here that there are pending pax?

Reply:
*sigh* yeah probably the same badge.

Reply:
Nope, that looks dumb and doesn't fit. highlight yellow?

Reply:
It looks so gross. What do

Reply:
I have settled on a font-awesome badge. Taking suggestions.